components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/TLConnectionStore.java

```diff
@@ -63,6 +63,10 @@ public static void commitAll() {
 		for (DataServiceConnection conn : conns.values()) {
 		    try {
 		        conn.commit();
+                if (log.isDebugEnabled()) {
+                    log.debug("Committing connection: " + conn.toString() + ",ThreadID - " +
```

Comment:
After a comma, you've to put a space before the next word, e.g. + ", ThreadId - "... Please fix in all the places.